split_settings/tools.py

```diff
@@ -83,7 +86,7 @@ def include(*args: str, **kwargs) -> None:  # noqa: WPS210, WPS231, C901
 
     for conf_file in args:
         saved_included_file = scope.get(_INCLUDED_FILE)
-        pattern = os.path.join(conf_path, conf_file)
+        pattern = os.path.join(conf_path, conf_file or '')
```

Comment:
Not the cleanest solution, mypy was throwing error here
```
error: Argument 2 to "join" has incompatible type "str | None"; expected "str"  [arg-type]
```

but it the include is not optional an `IsADirectoryError` is raised.